test/main/features/runtime-provider-management/openCodeWindowsNodeModulesJunction.test.ts

```
@@ -0,0 +1,212 @@
+import fs from 'node:fs';
+import os from 'node:os';
```

ESLint

[error] 'os' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] Remove this unused import of 'os'.

(sonarjs/unused-import)